hydra-node/test/Hydra/Chain/Direct/Contract/Contest.hs

```diff
@@ -189,13 +189,13 @@ genContestMutation
     , _utxo
     ) =
     oneof
-      [ SomeMutation Nothing MutateSignatureButNotSnapshotNumber . ChangeHeadRedeemer <$> do
+      [ SomeMutation (Just "signature verification failed") MutateSignatureButNotSnapshotNumber . ChangeHeadRedeemer <$> do
           mutatedSignature <- arbitrary :: Gen (MultiSignature (Snapshot Tx))
           pure $
             Head.Contest
               { signature = toPlutusSignatures mutatedSignature
               }
-      , SomeMutation Nothing MutateToNonNewerSnapshot <$> do
+      , SomeMutation (Just "signature verification failed") MutateToNonNewerSnapshot <$> do
```

Comment:
This is not what we expect as we actually give a `healthySignature (fromInteger mutatedSnapshotNumber)`

Reply:
Solved: 
  Before we were mutating the closedSnapshotNumber for a value lower than healthyClosedSnapshotNumber, but we needed to have it bigger than healthyContestSnapshotNumber